library/Model.re

```diff
@@ -0,0 +1,61 @@
+module type Config = {
+  type t;
+  let table: string;
+  let parseRow: array(string) => t;
+};
+
+module Make = (Config: Config) => {
+  open Ezpostgresql.Pool;
+  open Lwt_result.Infix;
+
+  let findOne = (~connection, ~clause=?, ()) => {
+    let query =
+      "select * from "
```

Comment:
I think we can probably use this https://ocaml.org/learn/tutorials/format.html to simplify our string interpolations, otherwise will be hard to scale all this haha

Reply:
only for io, but I'll create some helpers to help on this